packages/flutter_tools/test/application_package_test.dart

```diff
@@ -305,6 +306,49 @@ void main() {
       expect(iosApp, null);
     }, overrides: overrides);
   });
+
+  group('FuchsiaApp', () {
+    final Map<Type, Generator> overrides = <Type, Generator>{
+      FileSystem: () => MemoryFileSystem(),
+      Platform: _kNoColorTerminalPlatform,
+      OperatingSystemUtils: () => MockOperatingSystemUtils(),
+    };
+    testUsingContext('Error on non-existing file', () {
```

Comment:
nit: spaces between calls to `testUsingContext`

Reply:
Done